doc/lightning-listpeers.7.txt

```diff
@@ -0,0 +1,63 @@
+LIGHTNING-LISTPEERS(7)
+=======================
+:doctype: manpage
+
+NAME
+----
+lightning-listpeers - Command for returning data on connected lightning nodes.
+
+SYNOPSIS
+--------
+*listpeers* ['id']
+
+DESCRIPTION
+-----------
+The *listpeers* RPC command returns data on nodes in the Lightning Network
+connected to this node.
+
+Once a connection to another lightning node has been established, using the
+*connect* command, data on the node can be returned using *listpeers* and the
+'id' that was used with the *connect* command.
+
+Supplying 'id' will return only data on a connected lightning node with a
+matching 'id', if one exists.
+
+If no 'id' is supplied, then data on all connected lightning nodes is returned.
+
+RETURN VALUE
+------------
+On success, a "peers" object is returned, containing an array of 0 or more
+peer objects.
+
+Each peer object contains the following data:
+
+- 'id' : The unique id of the peer
+- 'connected' : A boolean value showing the connection status
+- 'netaddr' : An array of network addresses the node is listening on
+- 'globalfeatures' : Bit flags showing supported global features (BOLT #9)
+- 'localfeatures' : Bit flags showing supported local features (BOLT #9) 
+- 'global_features' : Duplicate of globalfeatures
```

Comment:
This field is deprecated; no need to mention it in the manpage.  Same for `local_features`.

Reply:
Thanks for the clarification, they've been removed